transmission_interface/package.xml

```diff
@@ -17,17 +17,18 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <depend>tinyxml</depend>
-  <depend>roscpp</depend>
   <depend>pluginlib</depend>
-  <build_depend>hardware_interface</build_depend>
+  <depend>roscpp</depend>
+  <depend>tinyxml</depend>
+
   <build_depend>cmake_modules</build_depend>
+  <build_depend>hardware_interface</build_depend>
 
-  <test_depend>rosunit</test_depend>
   <test_depend>resource_retriever</test_depend>
+  <test_depend>rosunit</test_depend>
 
   <export>
     <rosdoc config="rosdoc.yaml"/>
```

Comment:
I don't think rosdoc is being used anymore

Reply:
What do you mean? The autogenerated documentation is still being autogenerated: http://docs.ros.org/api/ and http://docs.ros.org/melodic/api/transmission_interface/html/c++/.

Are you maybe thinking of the rosdoc -> rosdoc_lite transition between Groovy and Indigo?

Reply:
You are right, for some reason I was under the impression these additional yaml files were ignored since Indigo.